src/index.html

```diff
@@ -22,6 +22,362 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="container">
+        <div class="header-img">
+          <a
+            href="#"
+            class="logo-img"
+          >
```

Comment:
Fix it everywhere if you have only 2 attributes
```suggestion
          <a href="#" class="logo-img">
```